challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It is a relative path because of the /
```

Comment:
Actually, it's an Absolute path. How can we tell? If we were to move from `~` to a different location in the filesystem, `/Users/blah_blah/Desktop` still points to the exact same place - the Desktop. Relative paths are paths that are based on _where **you** are_.